plugins/inputs/procstat/procstat.go

```diff
@@ -323,6 +323,10 @@ func (p *Procstat) updateProcesses(pids []PID, tags map[string]string, prevInfo
 			if name, _ := info.Name(); name == "" {
 				continue
 			}
+			if _, err := p.createProcess(pid); err != nil {
```

Comment:
Perhaps it would be better to call this [PidExists](https://pkg.go.dev/github.com/shirou/gopsutil@v3.21.6+incompatible/process#PidExists) function instead? It seems createProcess calls [NewProcess](https://pkg.go.dev/github.com/shirou/gopsutil@v3.21.6+incompatible/process#NewProcess) which seems to do the same thing but we don't need the resulting process structure at all.